### DISCUSSION TITLE
unclear how to tune dart parameters LightGBMTunerCV

### DISCUSSION THREAD

**Discussion by antonkulaga** [optuna/optuna#4196] [Q&A]
### Don't use GitHub Issues to ask support questions.

I checked API of LightGBMTunerCV and could not find how to specify dart as boosting type, also could not find tune_drop_out or similar function. Could you please clarify how to tune boosting type with LightGBMTunerCV ?

**Comment by gen740** [comment]
I guess you can use [`optuna.integration.lightgbm.train()`](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.integration.lightgbm.train.html#optuna.integration.lightgbm.train) and pass the params value, as [you do](https://github.com/microsoft/LightGBM/blob/master/examples/python-guide/simple_example.py#L25-L43) in `lightgbm.train` (of course if you want to use `lightgbm.cv()`, use `cv()`). Examples are [here](https://github.com/optuna/optuna-examples/blob/main/lightgbm/lightgbm_tuner_simple.py). 

Similarly, in `optuna.integration.lightgbm.LightGBMTunerCV`, in which `lightgbm.cv()` is called, you can pass these params by setting `LightGBMTunerCV`'s `params` arg.

I have no idea what you mean about `tune_drop_out`.
